WeakAurasOptions/LoadOptions.lua

```diff
@@ -510,7 +510,7 @@ function OptionsPrivate.ConstructOptions(prototype, data, startorder, triggernum
         if (arg.showExactOption) then
           options["exact"..name] = {
             type = "toggle",
-            width = WeakAuras.normalWidth - 0.1,
```

Comment:
I need to look at this again.

Reply:
It wasn't shown at the same place for all triggers (aura vs cast) and didn't have some position whither a scrollbar is shown or not
I made it always shown on the left of the input